apps/backend/api/models/Notification.js

```diff
@@ -8,6 +8,44 @@ import { en } from '../../lib/i18n/en'
 import { es } from '../../lib/i18n/es'
 const locales = { en, es }
 
+const DEFAULT_USER_AVATAR = 'https://www.gravatar.com/avatar'
+const DEFAULT_GROUP_AVATAR = '/default-group-avatar.svg'
+
+async function getIconUrl (notification) {
+  const { activity } = notification.relations
+  const reasons = activity.get('meta').reasons
+  const reason = Notification.priorityReason(reasons)
+  const groupRelatedReasons = [
+    'joinRequest',
+    'approvedJoinRequest',
+    'announcement',
+    'groupChildGroupInvite',
+    'groupChildGroupInviteAccepted',
+    'groupParentGroupJoinRequest',
+    'groupParentGroupJoinRequestAccepted',
+    'newPost',
+    'memberJoinedGroup'
```

Comment:
I think we should add mention, chat, comment, trackCompleted, trackEnrollment. Those all are within groups

Reply:
I actually dont think we should ever show a user's avatar, except maybe for a DM with just that person? but that might be worth discussing

Reply:
Is the mini-app icon over other icon a common pattern? Is that an iOS thing?